templates/secret-shared-password.yaml

```diff
@@ -1,6 +1,8 @@
 # Default password is "password".
 # Password specified in values or on command line overrides password currently in secret.
 # If no password is specified, password in secret is preserved.
+# If passwordSecretRef is defined this will be skipped entirely (useful for EC)
+{{- if .Values.passwordSecretRef }}
```

Comment:
Should this conditional be flipped to `if not` since we only want to create the secret if you haven't provided the override?

Reply:
correct